examples/contrib/cifar10/main.py

```diff
@@ -75,6 +75,50 @@ def training(local_rank, config):
     # Create trainer for current task
     trainer = create_trainer(model, optimizer, criterion, lr_scheduler, train_loader.sampler, config, logger)
 
+    # Finding optimal Learning Rate using FastaiLRFinder
+    if config["use_lr_finder"]:
+        lr_finder = FastaiLRFinder()
+        to_save = {"model": model, "optimizer": optimizer}
+        with lr_finder.attach(
+            trainer, to_save, output_transform=lambda output: output["batch loss"], start_lr=0.2
+        ) as trainer_with_lr_finder:
+            trainer_with_lr_finder.run(train_loader)
+        logger.info(f"\tOptimal Learning Rate: {lr_finder.lr_suggestion()}")
+        # Remove the attached lr_scheduler to create a new one
+        trainer.remove_event_handler(lr_scheduler, Events.ITERATION_STARTED)
+        # if config["with_clearml"]:
+        #     lr_finder_plot = lr_finder.plot()
+        #     task.logger.report_matplotlib_figure(
+        #         title="LRFinder plot", series="LR vs Loss", figure=lr_finder_plot
+        #     )
```

Comment:
Why this code is commented ?

Reply:
Doesn't work as expected yet, I'm trying to understand but in their docs they don't explain this part well